continuum/tasks/utils.py

```diff
@@ -24,3 +25,33 @@ def split_train_val(dataset: TaskSet, val_split: float = 0.1) -> Tuple[TaskSet,
     val_dataset = TaskSet(x_val, y_val, t_val, dataset.trsf, dataset.data_type)
 
     return train_dataset, val_dataset
+
+
+def concat(dataset: TaskSet, *others: TaskSet) -> TaskSet:
```

Comment:
pour le nom, concat -> concat_taskset? ou c'est inutile?

Reply:
Je pense que concat suffit, less is more :p